### ISSUE TITLE
don't downgrade weld when installing payara

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse-ansible#70]
custom weld version breaks payara 5

**Comment by donsizemore** [comment]
I did this in the 70_weld_version branch but I forget why we didn't merge it. If you want me to pick it up again I can test?

**Comment by pdurbin** [comment]
@donsizemore as we discussed in person yesterday, yes, I still want some sort of fix. Right now if you try to specify Payara 5 instead of Glassfish 4.1 in main.yml Payara won't even start because of changes to Weld. So when we use something other than Glassfish 4.1 or 4.x (whatever's easier) we should stop patching Weld. We want to be able to easily spin up instances running newer application servers so we can test more easily. Thanks!

**Comment by donsizemore** [comment]
@pdurbin I have unmerged branches to address all of this, but as we discussed it got put on hold and I don’t remember why. I’ll test and merge.

**Comment by pdurbin** [comment]
Working now. Here's the local change I made:

```
[root@ip-172-31-32-109 dataverse]# git diff
diff --git a/defaults/main.yml b/defaults/main.yml
index aa44d1c..f52fe16 100644
--- a/defaults/main.yml
+++ b/defaults/main.yml
@@ -70,9 +70,8 @@ dataverse:
     siteurl:
     timeout: 180
     root: /usr/local
-    dir: glassfish4 # change this for payara, etc.
-    zipurl: http://dlc-cdn.sun.com/glassfish/4.1/release/glassfish-4.1.zip
-    #zipurl: https://search.maven.org/remotecontent?filepath=fish/payara/distri
+    dir: payara5
+    zipurl: https://search.maven.org/remotecontent?filepath=fish/payara/distrib
   google_analytics_key:
   java:
     version: 1.8.0
[root@ip-172-31-32-109 dataverse]# 
```

server.log is in the payara5 directory: /usr/local/payara5/glassfish/domains/domain1/logs/server.log

I guess I'll also test the weld switch branch without the fix above to make sure there are no regressions, that Glassfish 4.1 still installs ok.

**Comment by pdurbin** [comment]
> I guess I'll also test the weld switch branch without the fix above to make sure there are no regressions, that Glassfish 4.1 still installs ok.

Yep, all still seems well with Glassfish 4.1. I merged pull request #73 . Thank you, @donsizemore !!
